apps/examples/wc-next13/pages/index.tsx

```diff
@@ -3,7 +3,7 @@ import Head from 'next/head';
 import { createComponent } from '@lit-labs/react';
 import { PieButton, BUTTON_SIZE, BUTTON_VARIANT } from '@justeattakeaway/pie-button';
 
-const Button = createComponent({
+const PieBtn = createComponent({
```

Comment:
Would `ButtonComponent` be a little better thinking about it now? As it's a result of the create component method and might read better?

Reply:
I personally feel like the word `component` is redundant. So it comes down to `Button` or `PieButton`, of which I think `button` is too generic/ambiguous :)